src/index.js

```diff
@@ -9,5 +9,5 @@
  */
 
 export {Multiselect} from './multiselect';
-export {MultiselectBlockDragger} from './multiselect_block_dragger';
+export {MultiselectDraggable} from './multiselect_draggable';
```

Comment:
Is there any possible use case for us to expose MultiselectDraggable, for better integration with other plugins? If not, let's not do that.